lib/arelastic/queries/bool.rb

```diff
@@ -1,12 +1,13 @@
 module Arelastic
   module Queries
     class Bool < Arelastic::Queries::Query
-      attr_accessor :must, :filter, :should, :must_not
-      def initialize(must: nil, filter: nil, should: nil, must_not: nil)
+      attr_accessor :must, :filter, :should, :must_not, :options
+      def initialize(must: nil, filter: nil, should: nil, must_not: nil, **options)
         @must     = must
         @filter   = filter
         @should   = should
         @must_not = must_not
+        @options  = options
       end
```

Comment:
I would rather remove the use of kwargs from this API. This addition does not support initialization with string keys.